CHANGELOG.md

```diff
@@ -3,6 +3,39 @@ Please note that this data model will now create a new transformation for **all*
 
 If you are not already a Quickstart Data Model user, you can find out more information [here](https://fivetran.com/docs/transformations/quickstart)!
 
+# dbt_salesforce v1.0.0 
+
+**📣 THIS IS A MAJOR PACKAGE RELEASE! 📣** More details below. 
+## 🚨 Breaking Changes 🚨:
```

Comment:
We will also want to highlight the breaking change in the removal of the tmp models from the source here.

Reply:
Updated